### PR TITLE
Create FAQ for C++ extensions

### PR DESCRIPTION
**Pull request by goldsborough**
@soumith

### CODE REVIEW

File: advanced_source/cpp_extension.rst [hunk 1]
```diff
@@ -1033,3 +1033,34 @@ mechanism as well as a motivation for using them. You can find the code examples
 displayed in this note `here
 <https://github.com/pytorch/extension-cpp>`_. If you have
 questions, please use `the forums <https://discuss.pytorch.org>`_.
+
+FAQ
+---
```
Comment (soumith) [inline advanced_source/cpp_extension.rst]: @goldsborough it feels like the FAQ should go as a note into the API documentation, or into the FAQ in the API documentation.

Comment (goldsborough) [reply advanced_source/cpp_extension.rst]: Ok

**Comment by goldsborough** [comment]
Can someone merge this? @soumith @JoelMarcey

**Comment by JoelMarcey** [comment]
@goldsborough Should we wait until we fix the tutorial build before merging?

**Pull request merged**